code/src/java/plugin/pretokens/test/PreSkillTester.java

```diff
@@ -167,10 +167,10 @@ else if (isType && !foundSkill)
 					else if ((percentageSignPosition >= 0))
 					{
 						List<Type> mockTypes = target.getTrueTypeList(true);
-						for (Type mockType : mockTypes)
+						for (Type ignored : mockTypes)
```

Comment:
Is this is a bug now semantically being hidden due to an ignore?

Reply:
Yeah the `mockType` or `ignored` being unused looks like a bug